posthog/migrations/0447_alter_integration_kind.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.2.14 on 2024-07-18 14:26
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0446_annotation_dashboard_alter_annotation_scope"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="integration",
+            name="kind",
+            field=models.CharField(
+                choices=[("slack", "Slack"), ("salesforce", "Salesforce"), ("hubspot", "Hubspot")], max_length=10
```

Comment:
If we need a migration for every integration, this might get annoying. 😅

Reply:
Yeah I don't really get why migrations are necessary given that it doesn't actually do anything DB level... I might just remove the choice the next time im here and have it as a simple string